packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -1090,3 +1091,12 @@ const findOldestCallEnded = (calls: { [key: string]: { endTime?: Date } }): stri
 function clearParticipantReactionState(callContext: CallContext, callId: string, participantKey: string): void {
   callContext.setReceivedReactionFromParticipant(callId, participantKey, null);
 }
+
+/* @conditional-compile-remove(reaction) */
+function isCapableToReact(capabilitiesFeature: CapabilitiesFeatureState | undefined): boolean {
+  return (
+    capabilitiesFeature !== undefined &&
+    (capabilitiesFeature.capabilities.useReactions.isPresent ||
+      capabilitiesFeature.capabilities.useReactions.reason === 'FeatureNotSupported')
```

Comment:
Should this be?
```suggestion
      capabilitiesFeature.capabilities.useReactions.reason !== 'FeatureNotSupported')
```

Reply:
Oh wait, is this saying "if the feature capability is not supported", i.e. we don't know the status of the capability for this feature?

Reply:
Yes. For that `FeatureNotSupported` case, it would be on by default. This would be the case for ACS group & Rooms calls.